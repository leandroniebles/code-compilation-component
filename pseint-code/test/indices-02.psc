funcion foo(a)
	Leer a;
FinFuncion
Proceso sin_titulo
	dimension a(10);
	foo(a);
FinProceso
