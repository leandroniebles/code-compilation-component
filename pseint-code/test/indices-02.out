=== Line 2: ExeError 200
